### PR TITLE
adjust csharp SDK generation for Microsoft.AVS

### PR DESCRIPTION
**Pull request by ctaggart**

### CODE REVIEW

**Comment by lirenhe** [comment]
@isra-fel, could you take a look at this PR?

File: specification/vmware/resource-manager/readme.csharp.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azur
 
 ```yaml $(csharp)
 csharp:
-  package-version: 0.3.0
+  package-version: 0.1.0
```
Comment (isra-fel) [inline specification/vmware/resource-manager/readme.csharp.md:8]: Just FYI that the final package version is determined by the `<Version>` tag in your C# project's `*.csproj` file, so updating here is not necessary.

**Review by isra-fel — APPROVED** [review]
LGTM

File: specification/vmware/resource-manager/readme.csharp.md (hunk 1 above)
Comment (ctaggart) [reply specification/vmware/resource-manager/readme.csharp.md:8]: Good to know. Thanks.

**Comment by ctaggart** [comment]
@isra-fel, thanks for approving. Do you have the permissions to merge it? I do not.

**Comment by isra-fel** [comment]
I don't have the permission to merge.

**Comment by lirenhe** [comment]
There is a conflict of readme.csharp.md, @ctaggart, could you help to resolve it?

**Comment by ctaggart** [comment]
@lirenhe, conflict has been fixed. The version was modified in #9820,  but I'm removing because for .NET, the version is taken from the csproj.

**Comment by ctaggart** [comment]
@lirenhe, please merge this.

**Review by lirenhe — APPROVED** [review]

**Pull request merged**